docs/models.txt

```diff
@@ -139,22 +139,27 @@ Interface committee {
 
 Interface meeting {
     id: number;
-    url_name: string; // For unique urls.
-    is_template: boolean; // Unique within a committee
-    enable_anonymous: boolean;
+    welcome_title: string;
+    welcome_text: HTML;
 
-    // Old "general_*" configs
+    // General
     name: string;
     description: string;
     location: string;
     start_time: datetime;
-    end_time: datetime;
-    welcome_title: string;
-    welcome_text: HTML;
+    end_time: datetime; // TODO: is this needed?
     custom_translations: JSON;
 
+    // System
```

Comment:
I'm currently not sure, if those settings should be editable in the settings page, or managed in some kind of committee-detail/meeting-list -> THis is mainly a UI decision, but for now please put them into the settings

Reply:
Will do